### DISCUSSION TITLE
How to pin to a remote kubo node?

### DISCUSSION THREAD

**Discussion by tornadocontrib** [ipfs/helia#498] [Q&A] [answered]
I saw documentation about pinning to other pinning services, but not the remote kubo node.

**Comment by 2color** [comment, marked as answer]
You can use the https://github.com/ipfs/js-kubo-rpc-client in order to control a remote Kubo node, however, it's worth noting that Kubo itself **does not** expose the [Pinning API](https://ipfs.github.io/pinning-services-api-spec/).
